dev-manuals/achievements/achievement-types.md

```diff
@@ -0,0 +1,41 @@
+# Achievement Types
+
+This file describes the different types of achievements and how they affect different player types.
+
+## Concept
+
+Different achievements influence players at a different strength, based on which player type they are. The nature of the task is relevant to which player types are best addressed.
+
+## Player Types
+
+There are four different player types, for further information about the characteristics of each type check out the [gamification](../gamification/README.md) concept.
+
+### Killer
+
+The killer compares himself with others and wants to be the best at all costs.  
+Typical achievements would be something like this: `Kill x enemies`, `Reach 1st place on the leaderboard`, ...  
+For Gamify-IT, achievements like `Reach 1st place on the leaderboard` or `Win x rounds of Towercrush` are possible.
+
+### Achiever
+
+The achiever wants to progress as fast as possible and doesn't really care about unnecessary side quests.  
+Typical achievements would be something like this: `Reach level x`, `Complete chapter x`, ...  
+For Gamify-IT, achievements like `Complete world x` or `Unlock all areas` are possible.
+
+### Explorer
+
+The explorer wants to explore the world and find hidden areas and items.  
+Typical achievements would be something like this: `Find x hidden objects`, ...  
+For Gamify-IT, achievements like `Find all easter eggs`, `Travel a distance of x tiles` or `Activate all teleporters` are possible.
+
+### Socializer
+
+The socializer wants to interact and work together with other players.  
+Typical achievements would be something like this: `Add x friends`, `Player x rounds of multiplayer`, ...  
+For Gamify-IT, achievements like `Player x rounds of Towercrush` or `Player x times with the same player` are possible.
+
+## Achievement goals
+
+Completely (not shown at all until unlocked) or partially hidden (only title visible until unlocked) achievements can be an additional motivation to play. This is especially true for explorers and achievers.  
```

Comment:
Also, I'm still missing whether the achievements will be made globally for **every** course, or if every course can have its own achievements.
My personal preference would be achievements per course.